internal/cmd/local/k8s/cluster.go

```diff
@@ -110,6 +116,74 @@ func (k *kindCluster) Exists(ctx context.Context) bool {
 	return false
 }
 
+// LoadImages pulls images from Docker Hub, and loads them into the kind cluster.
+// This is a best-effort optimization, which is why it doesn't an error;
+// it's possible that only some images will be loaded.
+// TODO this should probably take a context, and handle cancellation.
```

Comment:
Is this out-of-date? Looks like it does take a context.

Reply:
Removed, thanks.